Library/Homebrew/cmd/commands.rb

```diff
@@ -8,6 +8,19 @@ module Homebrew
   module_function
 
   def commands
+    valid_options = ["--quiet",
+                     "--include-aliases"].freeze
+    invalid_options_by_user = []
+    ARGV.options_only.each do |option|
+      invalid_options_by_user << option unless valid_options.include?(option)
+    end
+    unless invalid_options_by_user.empty?
+      odie <<-EOS.undent
+        Invalid option#{"s" if invalid_options_by_user.length>1} provided: #{invalid_options_by_user.join " "}
+        #{"Valid option#{"s" if valid_options.length>1}: #{valid_options.join " "}" unless valid_options.empty?}
```

Comment:
Please use `Formatter.pluralize(valid_options.length, "option")`. [See here](https://github.com/Homebrew/brew/blob/5ab904ed93eafdb7e87e509fa928854880653db1/Library/Homebrew/cmd/update-report.rb#L83) for another example.